index.html

```diff
@@ -3289,6 +3290,43 @@ <h2 id="resid_misc">Resource identifiers</h2><!-- originally this section has a
 		<p class="advisement">Specifications that define new syntax for URIs or contained within URIs MUST specify that characters outside the ASCII repertoire are <a>percent encoded</a> using the UTF-8 <a>character encoding</a>.</p>
 	</div>
 
+	<section id="idn" class="subtopic">
+		<h3>Domain names</h3>
+		
+		<aside class="links" id="links_lang_decl">
+		<p class="links_title">Useful background and overviews for this section</p>
+		<ul>
+		<li class="w3"><p class="link"><a href="https://www.w3.org/International/articles/idn-and-iri/index.en.html">An Introduction to Multilingual Web Addresses</a></p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc4690">Review and Recommendations for Internationalized Domain Names (IDNs)</a> [[RFC4690]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5890">Internationalized Domain Names for Applications (IDNA): Definitions and Document Framework</a> [[RFC5890]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5891">Internationalized Domain Names in Applications (IDNA): Protocol</a> [[RFC5891]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5892">The Unicode Code Points and Internationalized Domain Names for Applications (IDNA)</a> [[RFC5892]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5893">Right-to-Left Scripts for Internationalized Domain Names for Applications (IDNA)</a> [[RFC5893]]</p></li>
+		<li class="w3"><p class="link"><a href="https://unicode.org/reports/tr46/">UTS #46: Unicode IDNA Compatibility Processing</a> [[UTS46]]</p></li>
+		<li class="w3"><p class="link"><a href="https://url.spec.whatwg.org/#idna">IDNA</a> in URL Standard [[URL]]</p></li>
+		</ul>
+		</aside>
+
+		<!-- <div class="req" id="idna2003">
+			<p class="advisement">Specifications SHOULD refer to IDNA2008 (from [[RFC5890]] to [[RFC5893]]), instead of IDNA2003 (from [[RFC3940]] to [[RFC3942]]).</p>
+		</div> -->
+
+		<div class="req" id="idna_refer">
+			<p class="advisement">Specifications dealing with domains directly should refer to the <a href="https://url.spec.whatwg.org/#idna">IDNA algorithms</a> in [[URL]], instead of IDNA2008 (from [[RFC5890]] to [[RFC5893]]) or IDNA2003 (from [[RFC3940]] to [[RFC3942]]).</p>
+		</div>
+
+		<p>IDNA2008 and IDNA2003 do not match what's implemented in web browsers. Browsers use IDNA2008 by way of [[UTS46]].</p>
+		
+			<div class="req" id="punycode">
+<p class="advisement">Specifications SHOULD NOT refer to Punycode, especially as a type of string. The IDNA2008 terminology of "A-labels" and "U-labels", defined in RFC 5890, SHOULD be used.</p>
```

Comment:
Hmm no? They should use the URL terminology and algorithms. Punycode is completely abstracted away.

Reply:
Now the problem is that there are some specs that do use the term "Punycode" to refer to the ASCII-Compatible Encoding form. Any suggestions for which term in URL should be used? Such as:

* https://html.spec.whatwg.org/#email-state-(type=email)
* https://w3c.github.io/webappsec-csp/#framework-directive-source-list
* https://w3c.github.io/rdf-concepts/spec/#note-iris

Reply:
HTML is being fixed in https://github.com/whatwg/html/pull/10522.

CSP should reference URL's domain to ASCII.

RDF seems like it wants "domain to Unicode", but it also references IETF's URI/IRI so I'm not sure it can be helped.